harness/determined/pytorch/lightning/_adapter.py

```diff
@@ -142,6 +150,17 @@ def __init__(self, context: PyTorchTrialContext) -> None:
                 lm = mnist.LightningMNISTClassifier(lr=context.get_hparam('learning_rate'))
                 super().__init__(context, lightning_module=lm)
 
+        Arguments:
+            context (``PyTorchTrialContext``)
```

Comment:
Is it some convention to use `(` and `)` for both types and values to choose?

Reply:
I think of the constant values as union literal types. python type would shape it like `Union[Literal['x'], Literal['y']]`

Reply:
what do you suggest

Reply:
we follow [this convention](https://sphinx-rtd-tutorial.readthedocs.io/en/latest/docstrings.html) since the docstring will be used for the sphinx doc.

Reply:
I had just followed the pattern from pytorch_trial. this is helpful thanks. I'll update it

Reply:
Sorry about the confusion! I actually meant this:

```
opt_level (str, optional, default="O1"):  Pure or mixed precision optimization level.
                Accepted values are "O0", "O1", "O2", and "O3", explained in detail above.
```
This also works with sphinx docstring. But I sent a wrong link.